apps/nextjs-website/src/components/editorialComponents/BannerLink/BannerLink.tsx

```diff
@@ -0,0 +1,39 @@
+import { getIconFromName } from '@/helpers/getIconFromName';
+import { Box, Container, Stack, Typography, useTheme } from '@mui/material';
+
+import { BannerLinkProps } from '@pagopa/pagopa-editorial-components/dist/components/BannerLink';
+
+export const BannerLink = (props: BannerLinkProps) => {
+  const { theme, body, title, decoration } = props;
+  const { palette, spacing } = useTheme();
+
+  const backgroundColor =
+    theme === 'dark' ? palette.primary.dark : palette.primary.light;
+
+  const textColor = palette.primary.contrastText;
+  return (
+    <Box bgcolor={backgroundColor} component='section' sx={{ width: '100%' }}>
+      <Container>
+        <Stack gap={4} sx={styles.main}>
+          {getIconFromName({ type: decoration, fill: textColor, size: 60 })}
+          <Stack textAlign='center' gap={spacing(2)}>
+            <Typography color={textColor} variant='h6'>
+              {title}
+            </Typography>
+            <Typography color={textColor} variant='body2'>
+              <div dangerouslySetInnerHTML={{ __html: body }} />
+            </Typography>
```

Comment:
The following suggestion removes the error `Error: Hydration failed because the initial UI does not match what was rendered on the server.` 
```suggestion
            <Typography color={textColor} variant='body2' component='div'>
              <div dangerouslySetInnerHTML={{ __html: body }} />
            </Typography>
```